types/wicg-entries-api/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Project: https://github.com/WICG/entries-api
+// Definitions by: Henning Kasch <https://github.com/HenningCash>
+// The "File and Directory Entries API" is a non-standard API available in many browsers.
+// This specs are based on the draft available at https://wicg.github.io/entries-api/
+
+declare class BaseFileSystemEntry {
+    protected constructor();
+
+    readonly isFile: boolean;
+    readonly isDirectory: boolean;
+    readonly name: string;
+    readonly fullPath: string;
+    readonly filesystem: FileSystem;
+
+    getParent(successCallback?: FileSystemEntryCallback, errorCallback?: ErrorCallback): void;
+}
+
+type ErrorCallback = (err: DOMException) => void;
+type FileSystemEntriesCallback = (entries: FileSystemEntry[]) => void;
+type FileSystemEntryCallback = (entry: FileSystemEntry) => void;
+type FileCallback = (file: File) => void;
+
+interface FileSystemFlags {
+    readonly create?: false;
```

Comment:
Hmm, also - why are they hardcoded to `false`? Shouldn't they accept any booleans?

Reply:
Hmm, not sure if I'm interpreting the specs correctly:
```
dictionary FileSystemFlags {
    boolean create = false;
    boolean exclusive = false;
};
```
Does that means it's a `boolean` that defaults to `false` or a `boolean` thats always `false`? :thinking:

Reply:
Not sure where I got that `readonly` from...